src/SpeedyTransforms/spectral_transform.jl

```diff
@@ -85,6 +85,9 @@ struct SpectralTransform{
     eigenvalues⁻¹::VectorType           # = -1/(l*(l+1))
 end
 
+# add eltype 2024.12.12 by roc
```

Comment:
no need to add this. in a git repo you can always do `blame` and see when and who wrote it

Reply:
removed this